src/main/java/no/nav/tag/dittNavArbeidsgiver/services/altinn/MockServer.java

```diff
@@ -9,18 +9,17 @@
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
 import org.springframework.context.annotation.Profile;
-import org.springframework.http.HttpStatus;
 import org.springframework.stereotype.Component;
 
 import java.net.URL;
 
 import static com.github.tomakehurst.wiremock.client.WireMock.equalTo;
 import static java.nio.charset.StandardCharsets.UTF_8;
 
-@Profile("dev")
+@Profile({"dev"})
 @Slf4j
-@Component@ConditionalOnProperty(prefix = "mock", name = "enabled", havingValue = "true")
-
+@Component
+@ConditionalOnProperty(prefix = "mock", name = "enabled", havingValue = "true")
```

Comment:
Er det nødvendig å ha både `ConditionalOnProperty` og profilstyring?

Reply:
True, kan fjerne dev-profilen. Men trenger conditional for å kunne kjøre Spring-tester.

Reply:
Kunne eventuelt laget en egen profil for "test"

Reply:
har fjernet denne linjen nå, bruker bare dev profilen istedenfor